build/pipelines/azure-pipelines.release.yaml

```diff
@@ -4,13 +4,6 @@
 # Store and the Windows image. This pipeline relies on Microsoft-internal resources to run.
 #
 
-schedules:
-- cron: "0 7 * * *"
-  displayName: Daily midnight build
-  branches:
-    include:
-    - master
-
```

Comment:
Release pipeline should be triggered after ci-internal done, and all build/tests passed successfully, so disable the auto trigger here.

Reply:
It seems to be not necessary to remove this schedule. This schedule is for the master branch, and we should check the ci result before merge any PR into master.